src/regimes.ts

```diff
@@ -1,17 +1,13 @@
-import {
-  PrivacyRegime,
-  PrivacyRegimeEnum,
-} from '@transcend-io/airgap.js-types';
+import { PrivacyRegime } from '@transcend-io/airgap.js-types';
+import { settings } from './settings';
+import { COMMA_AND_OR_SPACE_SEPARATED_LIST } from './utils/comma-and-or-space-separated-list';
+
+const { regimePrecedence = 'GDPR LGPD CPRA CDPA CPA Unknown' } = settings;
```

Comment:
Do we not need to add `regimePrecedence` to be a new setting in the settings file?

Reply:
This only needs to be added to ag-types in the monorepo for backend integration.

Reply:
Can you add a Height task for that/make sure to add it as well?

Reply:
ya this should definitely be something backend-specified